<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [Suite]" guice-stage="DEVELOPMENT" verbose="0">
  <test thread-count="5" name="Test(failed)" verbose="0">
    <classes>
      <class name="com.actitime.testcases.TC2_CreateCustomerLogout">
        <methods>
          <include name="setup"/>
          <include name="createCustomer"/>
          <include name="teardown"/>
        </methods>
      </class> <!-- com.actitime.testcases.TC2_CreateCustomerLogout -->
      <class name="com.actitime.testcases.TC1_ActitimeLoginLogout">
        <methods>
          <include name="setup"/>
          <include name="loginlogout"/>
          <include name="teardown"/>
        </methods>
      </class> <!-- com.actitime.testcases.TC1_ActitimeLoginLogout -->
      <class name="com.actitime.testcases.TC3_CreateWorkLogout">
        <methods>
          <include name="setup"/>
          <include name="teardown"/>
          <include name="creatework"/>
        </methods>
      </class> <!-- com.actitime.testcases.TC3_CreateWorkLogout -->
    </classes>
  </test> <!-- Test(failed) -->
</suite> <!-- Failed suite [Suite] -->
